requirements/requirements.in

```diff
@@ -2,9 +2,9 @@
 # Only generic requirements for django-ansible-base (or the common feature) should be listed here.
 # if you are add a new feature which requires dependencies they should be in a separate requirements_<feature>.in file
 #
-cryptography
-Django>=4.2.5,<4.3.0
-djangorestframework
-django-crum
-django-split-settings
-inflection
+cryptography<=42.0.5
+Django>=4.2.5,<4.2.15
+djangorestframework<=3.15.1
+django-crum<=0.7.9
+django-split-settings<=1.2.0
```

Comment:
I guess it's a personal preference thing, but I don't feel like this needs to be so opinionated. That's what the updater script gets us.

For this, a pin of `<1.3` or `<2` might be reasonable. Not upgrading to the next _major_ version is probably generally reasonable. But not even that's necessary. You could just leave everything unpinned until there's an issue and comment what the problem is.

Reply:
@AlanCoding One of two separate things that sent me down this rabbit hole in the first place was that `social-auth-app-django` released a z-version (5.4.2) which changed a field and thus triggers a migration in DAB[authentication].

We have no idea if that migration would break a system package that pins to 5.4.1.